client/rest-high-level/src/test/java/org/elasticsearch/client/DataFrameRequestConvertersTests.java

```diff
@@ -147,6 +148,24 @@ public void testGetDataFrameTransformStats() {
 
         assertEquals(HttpGet.METHOD_NAME, request.getMethod());
         assertThat(request.getEndpoint(), equalTo("/_data_frame/transforms/foo/_stats"));
+
+        assertFalse(request.getParameters().containsKey("from"));
+        assertFalse(request.getParameters().containsKey("size"));
+
+        getStatsRequest.setPageParams(new PageParams(0, null));
+        request = DataFrameRequestConverters.getDataFrameTransformStats(getStatsRequest);
+        assertEquals("0", request.getParameters().get("from"));
```

Comment:
I find the following syntax more readable:
        assertThat(request.getParameters(), hasEntry("from", "0"));

WDYT?